src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -63,19 +66,15 @@ public static int Extract(HashAlgorithmName hashAlgorithmName, ReadOnlySpan<byte
             return hashLength;
         }
 
-        private static void Extract(HashAlgorithmName hashAlgorithmName, int hashLength, ReadOnlySpan<byte> ikm, ReadOnlySpan<byte> salt, Span<byte> prk)
-        {
-            Debug.Assert(HashLength(hashAlgorithmName) == hashLength);
-            int written = CryptographicOperations.HmacData(hashAlgorithmName, salt, ikm, prk);
-            Debug.Assert(written == prk.Length, $"Bytes written is {written} bytes which does not match output length ({prk.Length} bytes)");
-        }
-
         /// <summary>
         /// Performs the HKDF-Expand function
         /// See section 2.3 of <a href="https://tools.ietf.org/html/rfc5869#section-2.3">RFC5869</a>
         /// </summary>
         /// <param name="hashAlgorithmName">The hash algorithm used for HMAC operations.</param>
-        /// <param name="prk">The pseudorandom key of at least <see cref="HashLength"/> bytes (usually the output from Expand step).</param>
+        /// <param name="prk">
```

Comment:
These doc comments referred to a private method which got moved. I updated them to the documentation that is on docs.